src/freenas/usr/bin/truenas-developer-enable.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+
+import json
+import os
+import stat
+import sys
+from subprocess import run
+
+
+TO_CHMOD = ['apt', 'dpkg']
+EXECUTE_BITS = stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH
+
+def set_readwrite(entry):
+    if 'RO' not in entry['fhs_entry']['options']:
+        return
+
+    # There shouldn't be a legitimate reason to edit files in /conf
+    if entry['fhs_entry']['name'] == 'conf':
+        return
+
+    print(f'Setting readonly=off on dataset {entry["ds"]}')
+    run(['zfs', 'set', 'readonly=off', entry['ds']])
+
+
+def chmod_files():
```

Comment:
Why do we need 20 files of python code instead of 2 lines of bash code?

Reply:
My early morning thought (pre-coffee) was that users may want to enable apt, dpkg, and set readonly=off independently of installing the dev tools (idea being that keeping actions to absolute minimum will encourage tinkerers to do this rather than writing a separate how-to), which will make it easier to identify via custom ZFS property. Is there a particular reason this is unacceptable (other than using more lines of code)?

Reply:
Let's do it with subprocess call then. The less code the clearer in this case. Current python code forces all permissions to 0755 which is definitely not the same as `chmod +x`

Reply:
I changed the os.chmod call to do +x.